ui/app/mainui/activitycenter/views/ActivityNotificationUnknownGroupChatInvitation.qml

```diff
@@ -0,0 +1,43 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import shared 1.0
+import shared.panels 1.0
+import utils 1.0
+
+import "../controls"
+import "../panels"
+import "../stores"
+
+ActivityNotificationMessage {
+    id: root
+
+    messageDetails.messageText: "Invite to an unknown group"
```

Comment:
```suggestion
    messageDetails.messageText: qsTr("Invite to an unknown group")
```

Reply:
and, are you sure the wording is correct? I'd expect "Invitation to an unknown group" here

Reply:
Sure sure, I was looking around and I couldn't find any design :)